src/storiesStore/reducer.js

```diff
@@ -0,0 +1,18 @@
+const defaultState = {
+  stories: [],
+};
+
+export default function reducer(state = defaultState, action) {
+  switch (action.type) {
+    case 'RECEIVE_STORIES': {
+      return { ...state, stories: action.stories };
+    }
+    case 'SET_LOADING': {
+      const newState = { ...state, stories: [...state.stories] };
```

Comment:
You're not mutating stories. 
```suggestion
      const newState = { ...state };
```

Reply:
On it.